src/gui/size.ts

```diff
@@ -62,14 +62,15 @@ export const size = {
 		return (this.calendar_line_height + 2 * this.calendar_event_border + this.calendar_day_event_padding) * 4
 	},
 
+	calendar_days_header_height: 25,
 	calendar_hour_width: 80,
 	calendar_hour_width_mobile: 30,
 	calendar_event_margin: 6,
 	calendar_event_margin_mobile: 2,
 	calendar_event_border: 1,
 	calendar_day_event_padding: 2,
 	drawer_menu_width: 44,
-	first_col_min_width: 240,
+	first_col_min_width: 270,
```

Comment:
this affects all views and probably breaks some breakpoints somewhere (pun not intended). I think I would prefer if we changed it for the calendar view only.